webapp-django/crashstats/supersearch/models.py

```diff
@@ -38,22 +38,6 @@
 )
 
 
-class SuperSearchFieldsWithoutConfig(super_search_fields.SuperSearchFields):
-    """In the SuperSearchFields implementation class we know with confidence
-    we can just call
-    socorro.external.es.super_search_fields.SuperSearchFields.get() without
-    first having instanciated the base class with the necessary configuration
-    needed to reach Elasticsearch.
-
-    This way we can call `.get()` on an instance of this class, which
-    just returns the fields.
-    """
-
-    def __init__(self, *args, **kwargs):
-        # Deliberately don't do anything
-        pass
```

Comment:
This was a hack to get around using just the fields from `SuperSearchFields`. Now we have `SuperSearchFieldsData`, so we can just use that.